drafts/str4d-overwinter-activation/draft1.rst

```diff
@@ -0,0 +1,237 @@
+::
+
+  ZIP: ???
```

Comment:
@daira could you please assign a ZIP number?

Also, do we make this non-draft once the PR is merged, or merge as draft for now and wait until closer to 1.1.0 to move to being official? We've requested feedback from third parties by March 2nd.